addons/quickmount/CfgVehicles.hpp

```diff
@@ -9,7 +9,7 @@ class CfgVehicles {
         isGlobal = 1;
         isTriggerActivated = 0;
         isDisposable = 0;
-        icon = QPATHTOF(UI\Icon_Module_QuickMount_ca.paa);
+        icon = "a3\ui_f\data\IGUI\Cfg\Actions\Obsolete\ui_action_getin_ca.paa";
```

Comment:
Obsolete?

Reply:
The non obsolete ones are all for a specific seat `getincommander`, `getindriver`, `getingunner`, `getinpilot`, `getincargo`, so I'm using the generic "arrow in box" icon which is not used by the game anymore, but in the obsolete folder since 2014.

Reply:
i see, thanks for explaining :)